ksqldb-functional-tests/src/test/resources/sql-tests/query-upgrades/filters.sql

```diff
@@ -328,10 +328,9 @@ CREATE OR REPLACE TABLE b AS SELECT id, COUNT(*)
 ----------------------------------------------------------------------------------------------------
 --@test: add filter to Suppress
 --@expected.error: io.confluent.ksql.util.KsqlException
---@expected.message: Upgrades not yet supported for TableSuppress
+--@expected.message: Upgrades not yet supported for StreamWindowedAggregate
```

Comment:
I'm fine simply deleting this test rather than updating the expected error message. There's already another one which expects `Upgrades not yet supported for StreamWindowedAggregate`.

Reply:
Maybe @agavra can chime in if this test still adds value or not?

Reply:
yeah we can probably remove this test - the idea was that if you had a suppress it would fail regardless of what was being aggregated, but I don't think it adds much value since suppress only makes sense with windowed aggregates anyway (which we already test for above)

Reply:
Follow up PR to remove this test: https://github.com/confluentinc/ksql/pull/9308